backend/data/migrations/20190108144810_user.js

```diff
@@ -11,6 +11,8 @@ exports.up = function(knex, Promise) {
     table.text('address');
     table.string('role');
     table.string('photoUrl');
+    table.boolean('setting_text').defaultTo(false);
```

Comment:
What are these changes for?
Is it really necessary to change the user model for this?

Reply:
I would have to talk with Ronald, but I believe we did need to change the user model in order to keep track of communication preferences in terms of email and text.

Reply:
Do we want to give this option, given that we'll only get phone numbers by a fraction of customers?

Reply:
Thinking about this, it is probably fine.
We don't have the ability to communicate via text, so i don't think we should give the user the option on the frontend right?

Reply:
I suppose that works; it would be much simpler to maintain if we just had email comms available. Shall I take out all text-related functionality?